live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -233,3 +233,37 @@
   with_items:
     - { key: "net.ipv4.conf.all.promote_secondaries", value: "1" }
     - { key: "net.ipv4.conf.default.promote_secondaries", value: "1" }
+
+#
+# Configure command audit logging
+#
+# We want to record all commands executed on the appliance. Opt out for
+# setsid since all ExecuteUtils.execute wrap each call with setsid.
+#
+# Note we can't use the "service" module, as that fails when running
+# inside the chroot environment, so we enable auditd using "command".
+#
+- command: /bin/systemctl enable auditd.service
```

Comment:
let's use `systemctl` rather than `/bin/systemctl` to be consistent with other invocations of `systemctl` elsewhere in our ansible roles.

Reply:
Git it.  Thanks